src/index.html

```diff
@@ -15,8 +15,101 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="header__logo"
+      >
+        <img
+          src="../src/images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="navigation">
+        <ul class="navigation__list">
+          <li class="list__item">
+            <a
+              class="link is-active"
```

Comment:
And here `navigation__link`

Reply:
Olá Pedro, obrigada, alteração feita!

Reply:
[DEMO LINK](https://devmichellebarcelos.github.io/layout_moyo-header/)

[TEST REPORT LINK](https://devmichellebarcelos.github.io/layout_moyo-header/report/html_report/)

[ x ] Header height is set in 1 place (for the links)

[ x ] Content is vertically centered (for any header height)

[ x ] CSS is used to show all letters in Uppercase (don't type them in HTML)

[ x ] Logo is an image wrapped with a link

[ x ] CSS Variable is used for a blue color

[ x ] Pseudo-element is used for a blue line below the active link

[ x ] Code follows all the [Code Style Rules ❗️](https://github.com/mate-academy/layout_moyo-header/pull/checklist.md)